### PR TITLE
You no longer blugeon railings when unanchoring them.

### LINKED ISSUE #49965: Railings are bludgeoned when unfastening
## Round ID:
Round ID: [132255](https://scrubby.melonmesa.com/round/132255)
BYOND Version: 513.1508
Server revision compiled on: 
Master commit: 6a599a213b447d2b063d9b7e3e4f98955e2a30b1

## Testmerges:
The following pull requests are currently test merged:
#49344: '[READY] Space Dragon Rework' by IndieanaJones at commit 8416daa7c4

## Reproduction:

Click railing (e.g. ones on second floor of Donut) with wrench. There's also no progress bar, but unwrenching seems instantaneous so I don't know if there should be one.

### PR DESCRIPTION
**Pull request by nemvar**
## About The Pull Request

title
Fixes https://github.com/tgstation/tgstation/issues/49965

## Changelog
:cl:
fix: You can now anchor/unanchor railings without bludgeoning them.
/:cl:

### CODE REVIEW

**Review by Rohesie — APPROVED** [review]
Stinky author, but good PR.

File: code/game/objects/structures/railings.dm [hunk 1]
```diff
@@ -29,13 +29,16 @@
...
-	if(!(flags_1&NODECONSTRUCT_1))
-		if(I.tool_behaviour == TOOL_WRENCH)
-			to_chat(user, "<span class='notice'>You begin to [anchored ? "unfasten the railing from":"fasten the railing to"] the floor...</span>")
-			if(I.use_tool(src, user, volume = 75, extra_checks = CALLBACK(src, .proc/check_anchored, anchored)))
-				setAnchored(!anchored)
-				to_chat(user, "<span class='notice'>You [anchored ? "fasten the railing to":"unfasten the railing from"] the floor.</span>")
-			return
+///Implements behaviour that makes it possible to unanchor the railing.
+/obj/structure/railing/wrench_act(mob/living/user, obj/item/I)
+	. = ..()
```
Comment (Rohesie) [inline code/game/objects/structures/railings.dm]:
```suggestion
	. = ..()
	if(.)
		return
```

Comment (nemvar) [reply code/game/objects/structures/railings.dm]: I would argue that the places where this behaviour is implemented are actually doing it wrong.
Adding a component that adds tool behaviour should not block other pre-existing tool behaviours which is exactly what that line would do.

**Comment by Dennok** [comment]
Add ability to deconstruct it please.

**Comment by nemvar** [comment]
You have that already, just bludgeon them.
:eyes:

**Pull request merged**